erddap/templates/private/dataset.deployment.xml

```diff
@@ -309,16 +309,20 @@
         <addAttributes>
             <att name="ioos_category">Quality</att>
             <att name="long_name">{{dest_var_remaps.get(req_var, req_var).replace('_qc', '')}} Variable Quality Flag</att>
-            {% if '_FillValue' not in qc_var_types['gen_qc'][req_var] %}
-            <att name='_FillValue' type='byte'>-127</att>
+            {% if req_var not in qc_var_types['gen_qc'] %}
+            <att name="flag_values">0,1,2,3,4,5,6,7,8,9</att>
```

Comment:
Why do the `gen_qc` variables have a different syntax for `flag_values` when compared to the `qartod` variables?

`<att name="flag_values">0,1,2,3,4,5,6,7,8,9</att>` 
vs
`<att name="flag_values">1B, 2B, 3B, 4B, 9B</att>`